src/FieldWarning/Assets/Model/Game/Loading.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading;
+using UnityEngine;
+
+public abstract class Loading
+{
+    private static Mutex mut = new Mutex();
+    public double percent_done = 0;
+    public bool finished = false;
+    public bool started = false;
+    public string name;
```

Comment:
General trend here, try to make variables as private as possible, especially when they're not used by other classes anyway.

Reply:
Thanks I will keep that in mind. These unfortunately all need to be public as they provide information about state and what the loader is doing.